benefit/data/sheets/CaseEventToFields/CaseEventToFields-sandl-nonprod.json

```diff
@@ -23,5 +23,47 @@
     "PageID": "2.0",
     "ShowSummaryChangeOption": "Y",
     "PageLabel": "Amend Reason"
+  },
+  {
+    "LiveFrom": "04/07/2022",
+    "CaseEventID": "addCaseOutcome",
+    "CaseFieldID": "caseOutcome",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "1.0",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "04/07/2022",
+    "CaseEventID": "addCaseOutcome",
+    "CaseFieldID": "didPoAttend",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 2,
+    "PageID": "1.0",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "caseUpdated",
+    "CaseFieldID": "caseOutcome",
+    "CaseTypeID": "Benefit",
+    "LiveFrom": "05/07/2022",
+    "DisplayContext": "OPTIONAL",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 14,
+    "PageID": "1.0"
+  },
+  {
```

Comment:
I know its not necessary but you've left out "LiveFrom" here if you want to be consistent

Reply:
They were there just not at the beginning of the block. I have moved them to the beginning.